eva/catalog/catalog_utils.py

```diff
@@ -43,6 +50,15 @@ def get_video_table_column_definitions() -> List[ColumnDefinition]:
     return columns
 
 
+def get_table_primary_columns(table_catalog_obj: TableCatalogEntry):
+    if table_catalog_obj.table_type == TableType.VIDEO_DATA:
+        return get_video_table_column_definitions()[:2]
+    elif table_catalog_obj.table_type == TableType.IMAGE_DATA:
+        return get_image_table_column_definitions()[:1]
```

Comment:
There is a `get_image_table_primary_columns` below doing the same thing. Maybe remove it?